docs/run/run-jobs-batch.mdx

```diff
@@ -9,7 +9,11 @@ description: How to run quantum computing jobs in batch mode using Qiskit Runtim
 Batch mode can shorten processing time if all jobs can be provided at the outset. If you want to submit iterative jobs, use [sessions](sessions) instead.  Using batch mode has these benefits:
 
 -  The jobs' classical computation, such as compilation, is run in parallel. Thus, running multiple jobs in a batch is significantly faster than running them serially. 
--  There is no delay between jobs, which can help avoid drift.
+    <Admonition>
+    At most five jobs can be processed in parallel, which might include other users' jobs.
+    </Admonition>
```

Comment:
Perhaps we can combine this with the last sentence and say something like

"While your batch jobs will run as closely together as possible, they don't get exclusive access to the backend. Therefore, your batch jobs may run in parallel with other users' jobs if there is enough processing capacity on the system. Additionally, system calibration jobs could run between the batched jobs."